packages/product-feed-mergado/src/Model/FeedItem/MergadoFeedItemFactory.php

```diff
@@ -84,7 +84,7 @@ public function createForProduct(Product $product, DomainConfig $domainConfig):
             [],
             $this->availabilityFacade->isProductAvailableOnDomainCached($product, $domainId) ? 'in stock' : 'out of stock',
             $product->getBrand(),
-            $this->productUrlsBatchLoader->getResizedProductImageUrl($product, $domainConfig),
+            $this->productUrlsBatchLoader->getProductImageUrl($product, $domainConfig),
```

Comment:
alternative image still uses `?width` attribute.

Please verify also the other feeds

Reply:
Fixed